SP23.P02.Web/Controllers/StationsController.cs

```diff
@@ -74,8 +80,14 @@ public ActionResult<TrainStationDto> UpdateStation(int id, TrainStationDto dto)
             return NotFound();
         }
 
+        if (!User.IsInRole(Role.Admin))
```

Comment:
It should only forbid the User if they are not an admin *and* not the manager of this station. A simple way to check if the User is the manager if the station is to see if `User.Identity?.Name` and `station.Manager?.UserName` are equal.

This causes another one of the failed tests because, in the tests, "bob" is set to be one of the managers of one of the stations, and then the tests will check if "bob" is actually able to modify the station.